test/hotspot/gtest/gc/shenandoah/test_shenandoahNumberSeq.cpp

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
An earlier version of this test is in tip for an earlier bug fix. I am happy to consult if there is any confusion during a merge from tip. In this specific case, the contents of this file should take precedence.

Reply:
merged from master; resolving.